.github/workflows/setup_environment.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Install packages
+sudo snap install core
+sudo apt-get remove -qy lxd lxd-client
+sudo snap install lxd --channel=latest/stable
+sudo snap refresh lxd --channel=latest/stable
```

Comment:
Does this have any effect?

Reply:
I saw it in operator action .... not sure I believe this is preventing some weird snap install behavior